www/src/templates/Markdown/Markdown.tsx

```diff
@@ -1,24 +1,23 @@
-import { MDXRenderer } from 'gatsby-mdx';
-import { MDXProvider } from '@mdx-js/react'
-import { graphql } from 'gatsby';
+import {MDXRenderer} from 'gatsby-mdx';
```

Comment:
have we changed the liner settings? I thought preference was spaces before/after the braces @jennasalau

Reply:
@tilanka this is the preference, I added the Deloitte linter late last night because i was seeing lots of inconsistencies